### PR TITLE
Add pegasas to kubernetes-sigs org

### PR DESCRIPTION
**Pull request by pegasas**
I am an active member of `kubernetes org` and would like to apply to join the `kubernetes-sigs org`, and look forward to contributing more.

Eligibility to apply has been met as follows:

* Meet the conditions for [community membership](https://github.com/kubernetes/community/blob/master/community-membership.md#kubernetes-ecosystem)
* [REQUEST: New membership for @pegasas  #4405](https://github.com/kubernetes/org/issues/4405)

SIG projects I am involved with:
* https://github.com/kubernetes-sigs/gateway-api
* https://github.com/kubernetes-sigs/kueue

### CODE REVIEW

File: config/kubernetes-sigs/org.yaml [hunk 1]
```diff
@@ -734,6 +734,7 @@ members:
 - PatrickLang
 - pbnj
 - Pensu
+- pegasas
```
Comment (nikhita) [inline config/kubernetes-sigs/org.yaml:737]: @pegasas the list needs to be sorted

Comment (pegasas) [reply config/kubernetes-sigs/org.yaml]: Thanks @nikhita !
Done.

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

**Comment by pegasas** [comment]
Thank you very much! @nikhita

**Pull request merged**